aws_lambda_powertools/logging/logger.py

```diff
@@ -112,19 +110,48 @@ def __init__(
         self,
         service: str = None,
         level: Union[str, int] = None,
+        name: str = None,
         sampling_rate: float = None,
         stream: sys.stdout = None,
         **kwargs,
     ):
         self.service = service or os.getenv("POWERTOOLS_SERVICE_NAME") or "service_undefined"
+        self.name = name or self.service
```

Comment:
I'd have a slight preference to do something like `${name}.${self.service}`, seems easier from an experience point of view to me.

Reply:
The `dot` represents inheritance here, and there could be a limit on how many chars you could have in a Logger at some point, hence why either name or service. I actually had it like that but saw this quickly growing when tried something like `name=__name__` in a module